### ISSUE TITLE
Скласти таймлайн

### ISSUE THREAD

**Issue by KateF28** [KateF28/homepage#6]
Скласти список всіх (важливих) життєвих подій, визначних дат, досягнень, періодів, що дали приємний і неприємний досвід, тощо. В цьому конкретному випадку довгих списків не буває, натомість бувають погано згорнуті, тому пакуємо сюди все підряд, а про "лишні" події можна буде промовчати пізніше. Список повинен бути систематизованим і посортованим в такий спосіб, щоб читач швидко зрозумів "паттерн" і міг легко орієнтуватись в просторі і часі цього резюме.

**Comment by kateryna-fedorova** [comment]
```
<h3>Experience</h3>
<h6>Recent</h6>
<ul class="timeline">
	<li class="highlighted">
		<p><em>2018</em></p>
		<p>Amongst many fighters of the past was featured in the movie <strong><a href="https://www.imdb.com/title/tt1677720/" rel="external">"Ready Player One"</a></strong>. Ryu ready always!</p>
	</li>
	<li>
		<p><em>2017</em></p>
		<p>United with unknown until then to me Power Rangers to defea resurrected M.Bison. Damn good fighters they are, even not knowing karate ways!</p>
	</li>
	<li>
		<p><em>2016</em></p>
		<ul>
			<li>In <strong>"Street Fighter V"</strong> once and for all erased M.Bison from earth and tamed my inner Evil Ryu.</li>
			<li>Also spent some time on a side-project, mini-series <strong>"Street Fighter: Resurrection"</strong>.</li>
		</ul>
	</li>
</ul>
<h6>Back then</h6>
<ul class="timeline">
	<li>
		<p><em>2014</em></p>
		<p>Filmed in a martial arts tutorial <strong>"Street Fighter: Assassin's Fist"</strong>, revealing to audience forgotten art of Ansatsuken.</p>
	</li>
	<li>
		<p><em>2012</em></p>
		<p>The gods of intenet concluded me in the animated movie <strong>"Wreck-It Ralph"</strong>, but rewarded me with a good drink for it!</p>
	</li>
	<li>
		<p><em>2008</em></p>
		<p>My evil alter ego sneaked out of control and became a boss in <strong>"Street Fighter IV"</strong>.</p>
	</li>
	<li>
		<p><em>2006</em></p>
		<p>Borrowed my famous ripped gi to Soki in <strong>"Onimusha: Dawn of Dreams"</strong>, gods informed me he needed that for the battle.</p>
	</li>
</ul>
<h6>Way back then</h6>
<ul class="timeline">
	<li>
		<p><em>1998</em></p>
		<ul>
			<li>Defeated M. Bison in <strong>"Street Fighter Alpha 2"</strong>, bastard wanted to use my heroic body for his evil intentions, but retreated with shame.</li>
			<li>Also this year I encountered Marvel universe in <strong>"Marvel vs. Capcom: Clash of Super Heroes"</strong> fighter,loyal to my employer, Capcom, of course.</li>
		</ul>
	</li>
	<li>
		<p><em>1997</em></p>
		<p>Reunited with Ken, my friend and opponent in <strong>"Street Fighter III"</strong>. Men's friendship works mysteriously. Also helped with Arika project <strong>"Street Fighter EX"</strong></p>
	</li>
	<li>
		<p><em>1996</em></p>
		<p>Took care of villainous Akuma in <strong>"Street Fighter Alpha 2"</strong>. As a freelance on <strong>"Super Puzzle Fighter II Turbo"</strong> tried myself in a new genre, puzzle games. Let the wrath of seven heavens come upon these malicious enigmas, never again!</p>
	</li>
	<li>
		<p><em>1995</em></p>
		<p>Kept fighting in <strong>"Street Fighter Alpha: Warriors' Dreams"</strong>.</p>
	</li>
	<li>
		<p><em>1994</em></p>
		<p>Screened in a movie <strong>"Street Fighter"</strong>, so that the batllesof past would not be forgotten by millenial generation.</p>
	</li>
	<li>
		<p><em>1991</em></p>
		<p>Was promoted to <strong>"Street Fighter II"</strong>. Didn't stay there for long, celebrations and office dwelling is not for me!</p>
	</li>
	<li>
		<p><em>1987</em></p>
		<p>Started working for Capcom project <strong>"Street Fighter"</strong>. My partner was Ken Masters, the boss'name was Sagat.</p>
	</li>
</ul>
<h3>Publications</h3>
<ul class="timeline">
	<li>
		<p><em>2011</em></p>
		<p>The true and humble story of me under pseudonim Tyler Wilde in an article <strong>"The evolution of Ken and Ryu"</strong>.</p>
	</li>
	<li>
		<p><em>2003 - 2018</em></p>
		<p>For 15 years I have published comic books with Udon Entertainment depicting my adventures, bitter lessons and great victories.</p>
	</li>
</ul>
<h3>Achievements</h3>
<ul class="timeline">
<li>
	<ol class="inverted">
		<li>
			<p> <strong>#1</strong> in <strong>"Top 25 Street Fighter Characters"</strong> by IGN</p>
			<ul>
				<li class="details">
					<input type="checkbox" id="project-2">
					<label for="project-2" class="summary">IGN called me "fancy", "icon" and "a testament to the virtue of simplicity" &#10084;</label>
					<div class="labels">
						<a rel="external" class="label" href="https://www.google.com/search?q=fancy&rlz=1C1SQJL_ruUA839UA839&sxsrf=ALeKk03XZTUcOwSFiBSpl8b_Q9bimrjR2g:1588597590359&source=lnms&tbm=isch&sa=X&ved=2ahUKEwipwsGYo5rpAhVskIsKHbeKDLQQ_AUoAXoECB0QAw&biw=1707&bih=803&dpr=1.13">fancy</a>
						<a rel="external" class="label" href="https://www.google.com/search?q=icon&tbm=isch&ved=2ahUKEwjrm92Zo5rpAhUMCXcKHdkxDLQQ2-cCegQIABAA&oq=icon&gs_lcp=CgNpbWcQAzICCAAyAggAMgIIADICCAAyAggAMgIIADICCAAyAggAMgIIADICCABQ8tcBWPLXAWCE2gFoAHAAeACAAXmIAXmSAQMwLjGYAQCgAQGqAQtnd3Mtd2l6LWltZw&sclient=img&ei=WBOwXuuPN4yS3APZ47CgCw&bih=803&biw=1707&rlz=1C1SQJL_ruUA839UA839">icon</a>
						<a rel="external" class="label" href="https://www.google.com/search?q=a+testament+to+the+virtue+of+simplicity&tbm=isch&ved=2ahUKEwjhi_qno5rpAhXWzSoKHd4NBsIQ2-cCegQIABAA&oq=a+testament+to+the+virtue+of+simplicity&gs_lcp=CgNpbWcQA1CljQFYpY0BYPqOAWgAcAB4AIABXYgBXZIBATGYAQCgAQGqAQtnd3Mtd2l6LWltZw&sclient=img&ei=dhOwXuH4LNabqwHem5iQDA&bih=803&biw=1707&rlz=1C1SQJL_ruUA839UA839">a testament to the virtue of simplicity</a>
					</div>
				</li>
			</ul>
		</li>
		<li>
			<p> <strong>#2</strong> in <strong>"Top 20 Street Fighter Characters of All Time"</strong> by GameDaily, same council found me worth of <strong>#6</strong> place in <strong>"Top 25 Capcom Characters of All Time"</strong>, which I proudly share with my partner Ken</p>
		</li>
		<li>
			<p> I was named <strong>"5th Most Powerful Street Fighter Character"</strong> by Screen Rant-san</p>
		</li>
		<li>
			<p> <strong>#71</strong> in <strong>"Top 100 Heroes of All Time"</strong> by UGO Networks, also recognized <strong>#2</strong> in their list of <strong>"Top 50 Street Fighter Characters"</strong></p>
		</li>
		<li>
			<p> In a survey of 4000 online matches for <strong>Super Street Fighter IV</strong>, I got to be the most popular character, with 16.6% of players choosing my side</p>
		</li>
	</ol>
</li>
</ul>
<h3>Life & education</h3>
<ul class="timeline">
<li>
	<span class="dots">
		<span class="dot"></span>
		<span class="dot"></span>
		<span class="dot"></span>
	</span>
	<em>"To live is to fight and to fight is to live"</em>
	<p>I started learning fighting at my master Gouken's castle<sup><em>— I was beaten at head way too often to remember when exactly...</em></sup></p>
</li>
</ul>
```

**Comment by KateF28** [comment]
EXPERIENCE
2018 Amongst many fighters of the past was featured in the movie "Ready Player One". Ryu ready always!

2017
United with unknown until then to me Power Rangers to defea resurrected M.Bison.Damn good fighters they are, even not knowing karate ways!

2016
In "Street Fighter V" once and for all erased M.Bison from earth and tamed my inner Evil Ryu. Also spent some time on a side-project, mini-series "Street Fighter: Resurrection".

2014
Filmed in a martial arts tutorial "Street Fighter: Assassin's Fist", revealing to audience forgotten art of Ansatsuken.

2012 The gods of intenet concluded me in the animated movie "Wreck-It Ralph", but rewarded me with a good drink for it!

2008
My evil alter ego sneaked out of control and became a boss in "Street Fighter IV".

2006
Borrowed my famous ripped gi to Soki in "Onimusha: Dawn of Dreams", gods informed me he needed that for the battle.

1998
Defeated M. Bison in "Street Fighter Alpha 2", bastard wanted to use my heroic body for his evil intentions, but retreated with shame. Also this year I encountered Marvel universe in "Marvel vs. Capcom: Clash of Super Heroes" fighter, loyal to my employer, Capcom, of course.

1997
Reunited with Ken, my friend and opponent in "Street Fighter III". Men's friendship works mysteriously. Also helped with Arika project "Street Fighter EX"

1996
Took care of villainous Akuma in "Street Fighter Alpha 2". As a freelance on "Super Puzzle Fighter II Turbo" tried myself in a new genre, puzzle games. Let the wrath of seven heavens come upon these malicious enigmas, never again!

1995
Kept fighting in "Street Fighter Alpha: Warriors' Dreams".

1994 Screened in a movie "Street Fighter", so that the batllesof past would not be forgotten by millenial generation.

1991
Was promoted to "Street Fighter II". Didn't stay there for long, celebrations and office dwelling is not for me!

1987
Started working for Capcom project "Street Fighter". My partner was Ken Masters, the boss'name was Sagat.

PUBLICATIONS
2011 The true and humble story of me under pseudonim Tyler Wilde in an article "The evolution of Ken and Ryu"

2003-2018 For 15 years I have published comic books with Udon Entertainment depicting my adventures, bitter lessons and great victories.
